src/main/java/com/scrive/pdftools/tools/utils/FileInput.java

```diff
@@ -0,0 +1,22 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.itextpdf.text.pdf.codec.Base64;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+public class FileInput {
+
+    static public byte[] getFileContent(JSONObject obj) throws IOException {
+        String localFilePath = obj.optString("localFilePath");
+        String base64Content = obj.optString("base64Content");
+        if (base64Content != null && !base64Content.isEmpty()) {
+            return Base64.decode(base64Content);
+        } else {
+            try (RandomAccessFile f = new RandomAccessFile(localFilePath, "r")) {
```

Comment:
btw if we get to this point and `localFilePath` is null, this will fail. Maybe works if we handle in general all exceptions but maybe this method should perform some input validation?

Reply:
Yes. created a helper function, and validate both inputs.